### ISSUE TITLE
API Updates for Project Resources

### ISSUE THREAD

**Issue by acshields** [appirio-tech/connect-app#4202]
Comments from 11/24 discussion: 

- Resource API should be updated to look for Copilot's joining the Connect Project
- Add a Copilot role check to V5 API

@mtwomey @vikasrohit - I probably didn't capture this one correctly, so please leave comments below.

**Comment by vikasrohit** [comment]
@acshields @mtwomey these both items does not belong to projects app and I think they belong to https://github.com/topcoder-platform/resources-api and https://github.com/topcoder-platform/legacy-challenge-resource-processor. And I think we don't even need the first one i.e. `Resource API should be updated to look for Copilot's joining the Connect Project` because as @rootelement told we can not determine who is the actual copilot to the challenge when challenge is created.
`
However, there is one thing that fall under projects app which is to update the https://github.com/topcoder-platform/legacy-project-processor to use v5 resource api to add copilots instead of using v4.

**Comment by vikasrohit** [comment]
>However, there is one thing that fall under projects app which is to update the https://github.com/topcoder-platform/legacy-project-processor to use v5 resource api to add copilots instead of using v4.

@acshields @rootelement just realized that we even don't need to migrate to v5 resource api because we can not. Resource api works on challenge i.e. it adds resources to the challenge not to the project. v5 project api is already taking care of syncing back resources to legacy via the legacy-project-processor. So, we are just circling back to projects api itself. Nothing is to be done here.

**Comment by acshields** [comment]
So should this one be closed @vikasrohit ?

**Comment by vikasrohit** [comment]
Yes. Just wanted to hear from @rootelement if I missed something here.
